### PR TITLE
doc: 17.2.1 Release Notes

### PR DESCRIPTION
**Pull request by yuriw**
Fixes: https://tracker.ceph.com/issues/55974
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/releases/quincy.rst [hunk 1]
```diff
@@ -5,6 +5,114 @@ Quincy
...
+v17.2.1 Quincy
+==============
+
+This is the second stable release of Ceph Quincy.
+
+Notable Changes
+---------------
+* The "BlueStore zero block detection" feature (first introduced to Quincy in
+https://github.com/ceph/ceph/pull/43337) has been turned off by default with a
+new global configuration called `bluestore_zero_block_detection`. This feature,
```
Comment (anthonyeleven) [inline doc/releases/quincy.rst:17]: s/configuration/option/

We seem to mostly refer to tunables as "options"

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

File: doc/releases/quincy.rst (hunk 1 above)
Comment (idryomov) [reply doc/releases/quincy.rst:17]: There is also a `Bullet list ends without a blank line; unexpected unindent.` sphinx warning here -- the body needs to be indented.

File: doc/releases/quincy.rst [hunk 2]
```diff
@@ -5,6 +5,129 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.1 Quincy
+==============
+
+This is the second stable release of Ceph Quincy.
```
Comment (djgalloway) [inline doc/releases/quincy.rst:11]: Do we want to say "second stable release" or "first backport release"

Comment (jdurgin) [reply doc/releases/quincy.rst:11]: "first bugfix release" like we used for 16.2.1 sounds better to me (https://github.com/ceph/ceph/blob/255b0204d27d4747ef336e26c787e3b576f1ca8b/doc/releases/pacific.rst#v1621-pacific)

**Review by jdurgin — APPROVED** [review]

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
Suggest addressing the sphinx error then it's gtg

**Comment by djgalloway** [comment]
> Suggest addressing the sphinx error then it's gtg

Sorry, which error?

**Pull request merged**